### PR TITLE
practice for chart

### PR DESCRIPTION
**Pull request by 0xAlison**
I add some more practical features to the tree.html (cater to the topic for "exchanged students")
you can interact with the chart(e.g. if you click on the "europe" you'll see the listed european countries like France, Germany, etc)
image

### CODE REVIEW

File: src/main/webapp/tree.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                ['Australia', 'Global',             0,                               0],
+                ['Africa',    'Global',             0,                               0],
+                ['Brazil',    'America',            11,                              10],
+                ['USA',       'America',            52,                              31],
+                ['Mexico',    'America',            24,                              12],
+                ['Canada',    'America',            16,                              -23],
+                ['France',    'Europe',             42,                              -11],
+                ['Germany',   'Europe',             31,                              -2],
+                ['Sweden',    'Europe',             22,                              -13],
+                ['Italy',     'Europe',             17,                              4],
```
Comment (Sinyu104) [inline src/main/webapp/tree.html:23]: Since all the data are static, it can't be changed automatically.
Should it be auto-changing data?

**Review by Sinyu104 — APPROVED** [review]
Do we need to get data from our database?
or just remain static?

File: src/main/webapp/tree.html (hunk 1 above)
Comment (0xAlison) [reply src/main/webapp/tree.html:23]: yes^^ the chart part(first week practice) is only for hard-code(meaning static)
in the second week of the chart practice, we can read csv or other data form to import the data:)

Comment (Sinyu104) [reply src/main/webapp/tree.html:23]: Thanks, Alison!
I understand.

**Pull request merged**